src/main.rs

```diff
@@ -86,16 +86,6 @@ on what should be in that file.");
         None => PathBuf::new(),
     };
 
-    let data_path = match xdg_dirs.create_data_directory(&data_path) {
```

Comment:
If I'm not mistaken, xdg_dirs.create_data_directory either creates the directory given with &data_path, or a Subdirectory in XDG_DATA_HOME_DIR if data_path isn't a full path yet (something like --output ak2 will create the directory XDG_DATA_HOME_DIR/antikoerper/ak2). In other words, without deleting it, data_path is never empty when given to conf::load (except for errors maybe). Let's talk about this later.